src/lib/client.js

```diff
@@ -8,14 +8,15 @@ const omitUndefined = require('lodash/fp/omitBy')(isUndefined)
 const EventEmitter = require('eventemitter2')
 const notUndefined = require('lodash/fp/negate')(isUndefined)
 const startsWith = require('lodash/fp/startsWith')
-const getQuote = require('./util').getQuote
+const debug = require('debug')('ilp-core')
 
 class Client extends EventEmitter {
   /**
    * @param {Object} pluginOpts options for the ledger plugin
    * @param {Function} pluginOpts._plugin A ledger plugin constructor
    * @param {Object} [_clientOpts]
-   * @param {URI[]} [_clientOpts.connectors] A list of connectors to quote from
+   * @param {String[]} [_clientOpts.connectors] A list of connectors to quote from
+   * @param {Integer} [_clientOpts.messageTimeout] The number of milliseconds to wait for a response to sendMessage.
```

Comment:
The timeout isn't for the `POST /messages`; it's how long to wait for a response to a `quote_request` (or any other message that expects another message to be sent back in reply).